target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -190,10 +190,19 @@ TARGET_DEVICES += ubnt_litebeam-ac-gen2
 define Device/ubnt_nanobeam-ac
   $(Device/ubnt-wa)
   DEVICE_MODEL := NanoBeam AC
+  DEVICE_VARIANT := Gen1
```

Comment:
You should also do the same change to the Gen1 DTS model property, i.e. add "Gen1" there.